libraries/AC_AutoTune/AC_AutoTune_FreqResp.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+/*
+ Gain and phase determination algorithm
+*/
+
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Math/AP_Math.h>
+
+#define AUTOTUNE_DWELL_CYCLES                6
+
+class AC_AutoTune_FreqResp {
+public:
+    // Constructor
+    AC_AutoTune_FreqResp()
+{
+    // initialize test variables
+    meas_peak_info_buffer = new ObjectBuffer<peak_info>(AUTOTUNE_DWELL_CYCLES);
```

Comment:
not a big deal of course but I'd probably move the implementation of the constructor to the .cpp file.

Reply:
Can't move out of the header as the heli subclass uses the define.  Should I make an accessor for the subclass since this is a standalone.  I guess it seems odd that the subclass is using the define directly

Reply:
ok, if the heli subclass uses it then leaving it in the .h seems OK to me.